tools/test-proxy/Azure.Sdk.Tools.TestProxy/Properties/launchSettings.json

```diff
@@ -10,7 +10,6 @@
   "profiles": {
     "Azure.Sdk.Tools.TestProxy": {
       "commandName": "Project",
-      "commandLineArgs": "--help",
```

Comment:
I removed this to be able to debug. I'm not sure if it is intentional that it is included.

Reply:
Definitely unintentional. Thanks Josh.